connector/spanmetricsconnector/connector.go

```diff
@@ -520,11 +522,27 @@ func (p *connectorImp) buildAttributes(serviceName string, span ptrace.Span, res
 	if !contains(p.config.ExcludeDimensions, statusCodeKey) {
 		attr.PutStr(statusCodeKey, traceutil.StatusCodeStr(span.Status().Code()))
 	}
+
 	for _, d := range dimensions {
 		if v, ok := utilattri.GetDimensionValue(d, span.Attributes(), resourceAttrs); ok {
 			v.CopyTo(attr.PutEmpty(d.Name))
 		}
 	}
+
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScopeNameKey) && instrumentationScope.Name() != "" {
+		attr.PutStr(instrumentationScopeNameKey, instrumentationScope.Name())
+	}
+	if !contains(p.config.ExcludeInstrumentationScope, instrumentationScopeVersionKey) && instrumentationScope.Version() != "" {
+		attr.PutStr(instrumentationScopeVersionKey, instrumentationScope.Version())
+	}
+
+	attrs := instrumentationScope.Attributes()
+	for _, p := range p.config.InstrumentationScope {
```

Comment:
why we need to add this loop?

Reply:
I'am a little confuse, In this PR ,we only need add two spanmetrics labels: `span.instrumentation.scope.name`, `span.instrumentation.scope.version`  right?

Reply:
Sorry but I misunderstood something from this comment: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23662#issuecomment-1777451635
And thought we wanted to add all the attributes.

Sorry for that. Fixing...